packages/design-tokens/src/tokens/functional/components/bento/base.json

```diff
@@ -0,0 +1,43 @@
+{
+  "brand": {
+    "Bento": {
+      "gap": {
+        "large": {
+          "value": "var(--base-size-32)"
+        },
+        "medium": {
+          "value": "var(--base-size-24)"
+        },
+        "small": {
+          "value": "var(--base-size-20)"
+        }
+      },
+      "Item": {
```

Comment:
Can we lowercase the Item and Content please? It's only the pattern name that should be pascal cased.